### PR TITLE
update the deployement doc

### PR DESCRIPTION
**Pull request by syan10**
Signed-off-by: Yan, Shaopu <shaopu.yan@intel.com>

### CODE REVIEW

File: docs/deployment-instructions.md [hunk 1]
```diff
@@ -10,38 +10,71 @@ This below diagram depicts the high-level overview of the eHSM-KMS in K8S cluste
 ---
 
 ## Prerequisites
-* Ensure you already have a running kubenetes cluster environment, if not, please follow [k8s-setup-guide](k8s-setup-guide.md) to setup the K8S cluster.
-* Ensure you already have a NFS server, if not, please follow [nfs-setup-guide](nfs-setup-guide.md) to setup a nfs server.
+* A running kubenetes cluster environment, if not, please follow [k8s-setup-guide](k8s-setup-guide.md) to setup the K8S cluster.
+    * The worker nodes MUST support SGX(need FLC support for DCAP remote attestation).
+    * The Master is a generic node.
+* A centralized dkey-provisioning server node.
```
Comment (yang8621) [inline docs/deployment-instructions.md:16]: dkey-provisioning refer to dkeyserver ?

File: docs/deployment-instructions.md [hunk 2]
```diff
@@ -10,38 +10,71 @@ This below diagram depicts the high-level overview of the eHSM-KMS in K8S cluste
...
+    * MUST support SGX (w/ FLC) OR directly connect with a dedicated HSM.
```
Comment (yang8621) [inline docs/deployment-instructions.md:17]: Do we need to mention dedicated HSM?

File: docs/deployment-instructions.md (hunk 1 above)
Comment (syan10) [reply docs/deployment-instructions.md:16]: yes

File: docs/deployment-instructions.md (hunk 2 above)
Comment (syan10) [reply docs/deployment-instructions.md:17]: our reference solution is target for enclave,  but better to mention it as another choice.

File: docs/deployment-instructions.md [hunk 3]
```diff
@@ -10,38 +10,71 @@ This below diagram depicts the high-level overview of the eHSM-KMS in K8S cluste
...
+
 
     nfs:
         path: /nfs_ehsm_db                     --> <your_nfs_folder>
-        server: 1.2.3.4                        --> <your_nfs_ipaddr>
+        server: 1.2.3.4                        --> <your_nfs_ip>
+
+     containers:
+      - name: dkeycache
+        image: intel/ehsm_dkeycache:latest     --> <your_dkeycache_image>
```
Comment (yang8621) [inline docs/deployment-instructions.md:59]: intelccc  and some below

File: docs/deployment-instructions.md [hunk 4]
```diff
@@ -10,38 +10,71 @@ This below diagram depicts the high-level overview of the eHSM-KMS in K8S cluste
 ---
 
 ## Prerequisites
-* Ensure you already have a running kubenetes cluster environment, if not, please follow [k8s-setup-guide](k8s-setup-guide.md) to setup the K8S cluster.
-* Ensure you already have a NFS server, if not, please follow [nfs-setup-guide](nfs-setup-guide.md) to setup a nfs server.
+* A running kubenetes cluster environment, if not, please follow [k8s-setup-guide](k8s-setup-guide.md) to setup the K8S cluster.
```
Comment (HawieLong) [inline docs/deployment-instructions.md:13]: kubernetes..

File: docs/deployment-instructions.md [hunk 5]
```diff
@@ -10,38 +10,71 @@ This below diagram depicts the high-level overview of the eHSM-KMS in K8S cluste
...
+
+    # run the script
+    ./run_dkeyserver.sh
+    ```
+    ![dkeyserver_status.png](./diagrams/dkeyserver_status.PNG)
+
+* Download and modify the yaml file in the K8S master node.
     ```Shell
-    $ wget https://github.com/intel/ehsm/tree/main/ehsm_kms_service/k8s/ehsm-kms.yaml
+    Notes: You can get the YAML file from:
```
Comment (HawieLong) [inline docs/deployment-instructions.md:42]: example YAML file?

**Review by yang8621 — APPROVED** [review]

**Pull request merged**